code/datums/skills/balling.dm

```diff
@@ -0,0 +1,6 @@
+/datum/skill/balling
+	name = "Balling"
+	title = "Baller"
+	desc = "Fuck it, we ball."
```

Comment:
Make this another reference to space jam or hoopz barkley instead of just being vulgar

Reply:
No

Reply:
Sorry thats a bit rude but your comment kind of pissed me off. I don't know why anyone would care about what words I use for the description of a silly skill in a game that features some of the most barbaric shit imaginable. 

If a maint actually cares I'll change it.

Reply:
I don't mind vulgarity if used well but it just seems a little bit in poor taste. Is it a reference or something? I just feel like you could have a better description, I could have worded it less sternly, my bad.

Reply:
Fuck it we ball is a meme

Reply:
Memes just aren't as good as they used to be I guess...